setup.cfg

```diff
@@ -17,11 +17,11 @@ classifiers =
     Operating System :: POSIX
     Programming Language :: Python :: 3
     Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.10
     Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
-    Programming Language :: Python :: 3.10
```

Comment:
Updating [`setup-cfg-fmt` to version 1.17.0](https://github.com/pytest-dev/pytest/blob/6.2.x/.pre-commit-config.yaml#L42) in our pre-commit config (as on main) will fix this.